packages/plugins/integration-make-com/src/lib/make-com.controller.ts

```
@@ -49,7 +52,76 @@ export class MakeComController {
+		description: 'Tenant ID not found in request context'
+	})
+	@Post('/oauth-settings')
+	async updateOAuthSettings(@Body() input: UpdateMakeComSettingsDTO): Promise<IMakeComIntegrationSettings> {
+		// Validate that both clientId and clientSecret are provided together
+		if ((input.clientId && !input.clientSecret) || (!input.clientId && input.clientSecret)) {
+			throw new Error('Both clientId and clientSecret must be provided together');
+		}
+
+		// Update environment variables with provided OAuth settings if they exist
+		if (input.clientId && input.clientSecret) {
+			let clientId = this._config.get<string>('makeCom.clientId');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'clientId' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)